### ISSUE TITLE
Need to include .babelrc

### ISSUE THREAD

**Issue by jladuval** [philcockfield/ui-harness#97]
Pretty minor issue but I didn't realise ui-harness needed a .babelrc in the root directory to work. Without it I had issues compiling jsx. Pretty easy to replicate, just download the ui-harness-sample and remove the .babelrc file. Our babel config is part of our webpack config so it can't be assumed that everyone will have the .babelrc file present.

**Comment by frederickfogerty** [comment]
Yep, as you said we were operating on the false assumption that people had a `.babelrc` present already. We should add a check for this in the `postinstall` step, and add one if there isn't one present.

**Comment by frederickfogerty** [comment]
Alternatively we could put in our webpack config, thus not affecting the user's existing babel config
